sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -132,6 +132,14 @@ class RuntimeConfig private[sql](sqlConf: SQLConf = new SQLConf) {
     sqlConf.unsetConf(key)
   }
 
+  /**
+   * Indicates whether the configuration property with the given key
+   * is modifiable in the current session.
```

Comment:
nit: If a key is not a valid conf, this function will still return false. Should we specify this in the comments?

Reply:
I added `@return` tag with this info.